Examples/Swift/ViewController.swift

```diff
@@ -165,34 +165,36 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         guard let route = self.currentRoute else { return }
 
         exampleMode = .styled
-
-        let navigationViewController = NavigationViewController(for: route, locationManager: locationManager())
-        navigationViewController.navigationDelegate = self
         
-        // Set a custom style URL
-        navigationViewController.mapView?.styleURL = URL(string: "mapbox://styles/mapbox/navigation-guidance-day-v1")
-
-        let style = Style()
+        let style = DefaultStyle()
```

Comment:
I think we will maintain at least two styles in the near future, one for landscape and one for portrait (or the corresponding size classes). The example is a quick and dirty approach. I imagine you would subclass DefaultStyle either just once for a generic style or let's say per profile identifier MyBikeStyle or per size class MyCompactStyle.